secure/key-manager/src/tests.rs

```diff
@@ -420,8 +443,10 @@ fn verify_ability_to_read_move_data<T: LibraInterface>(node: Node<T>) {
 }
 
 #[test]
-// This tests that a manual consensus key rotation can be performed by generating a new keypair,
-// creating a new rotation transaction, and executing the transaction locally.
+// This tests that a manual consensus key rotation can be performed (in the event of an unexpected
+// failure). To do this, the test generates a new keypair locally, creates a new rotation
+// transaction manually, executes the transaction on-chain and then updates secure storage
+// manually by overwriting the consensus key with the newly generated one.
 fn test_manual_consensus_rotation() {
     // Test the mock libra interface implementation
```

Comment:
these should be two different tests

Reply:
Thanks! I've removed the second half of the test after seeing your comment below. See my response below.